espressif/vscode-esp-idf-extension
# remove ccpp extension dependency

## Description

Remove Microsoft C/C++ extension from vscode extension dependencies. This will simplify release flow for vsix for open-vsx   and vscode marketplace release. Allow user to define their own language extensions since we don't provide support for them.

Fix #672 

## Type of change

- New feature (non-breaking change which adds functionality)

## Steps to test this pull request

Test `yarn package` in CLI to try to build this extension. Follow Build from source or CONTRIBUTING.md tutorial. Observe Microsoft C/C++ extension is not added automatically as before.

- Expected behaviour: No C/C++ extension dependency required.

- Expected output:

## How has this been tested?

Manual testing in macOS build from source.

**Test Configuration**:
* ESP-IDF Version: Irrelevant
* OS (Windows,Linux and macOS): macOS

## Dependent components impacted by this PR:

- Microsoft C/C++ extension

## Checklist
